logging/config.go

```diff
@@ -113,15 +114,13 @@ func newLoggerFromConfig(configJSON string, levelOverride string, opts []zap.Opt
 }
 
 func zapConfigFromJSON(configJSON string) (*zap.Config, error) {
-	if configJSON == "" {
```

Comment:
Interesting -- this changes the behavior to explicitly make an empty config okay.

I like that.

Reply:
Thanks! Previously on parsing the configmap, an empty config would be replaced with the defaultZLC JSON and later stages would require the config JSON to be present. I've changed this so that an empty config in the configmap is parsed to an empty config which later causes the default production logger to be used.

Reply:
Do you mean no logging configmap?